inc/class-wp-forms-api.php

```diff
@@ -910,11 +916,31 @@ static function process_element( $element, &$values, &$input ) {
 			else if( isset( $element['#type'] ) && $element['#type'] != 'composite' ) {
 				$element['#value'] = sanitize_text_field( $element['#value'] );
 			}
+
+			// Validate element value and place result in #valid
+			if( !empty( $element['#validator'] ) ) {
+				$validators = (array) $element['#validator'];
```

Comment:
Hm, that's a good point. I'll update it to check callability first before iterating.